docs/source/contributing/testing.rst

```diff
@@ -167,69 +166,44 @@ For example, to test the ``Circle`` VMobject which resides in
+  def test_circle(scene):
+      circle = Circle()
+      scene.play(Animation(circle))
 
-.. note:: 
+The decorator can be used with or without parentheses. **By default, the test only tests the last frame. To enable multi-frame testing, you have to set ``last_frame=False`` in the parameters.**.
 
-   If the file already exists, edit it and add the test within the file. The 
-   ``Scene`` will be tested thanks to the ``GraphicalUnitTester`` that lives
-   in ``tests/utils/GraphicalUnitTester.py``. Import it with ``from
-   ..utils.GraphicalUnitTester import GraphicalUnitTester``.
+.. code:: python
+  @frames_comparison(last_frame=False)
+  def test_circle(scene):
+      circle = Circle()
+      scene.play(Animation(circle))
 
-To test all the scenes in the module, we do the following:
+You can also specify, when needed, which base scene you need (ThreeDScene, for example) : 
 
 .. code:: python
+  @frames_comparison(last_frame=False, base_scene=ThreeDScene)
+  def test_circle(scene):
+      circle = Circle()
+      scene.play(Animation(circle))
 
-    @pytest.mark.parametrize("scene_to_test", get_scenes_to_test(__name__), indirect=False)
-    def test_scene(scene_to_test, tmpdir, show_diff):
-        GraphicalUnitTester(scene_to_test[1], MODULE_NAME, tmpdir).test(show_diff=show_diff)
+Feel free to check the documentation of ``@frames_comparison`` for more.
 
-The first line is a `pytest decorator
-<https://docs.pytest.org/en/stable/parametrize.html>`_.
-It is used to run a test function several times with different
-parameters. Here, we pass in all the scenes as arguments.
+Note that tests name must follow the syntax ``test_<thing_to_test>``, otherwise pytest will not recognize it as a test.
 
 .. warning::
   If you run pytest now, you will get a ``FileNotFound`` error. This is because
   you have not created control data for your test. 
 
-Next, we'll want to create control data for ``CircleTest``. In
-``tests/template_generate_graphical_units_data.py``, there exists the
-function, ``set_test_scene``, for this purpose.
-
-It will look something like this:
-
-.. code:: python
-
-    class CircleTest(Scene):
-        def construct(self):
-            circle = Circle()
-            self.play(Animation(circle))
-
-
-    set_test_scene(CircleTest, "geometry")
-
-``set_test_scene`` takes two parameters: the scene to test, and the
-module name. You can generate the test data by running the file (it suffices to type the name of the file in the terminal; you do not have to run
-it like how you would normally run manim files). It will automatically generate the control data in the
-right directory (in this case,
-``tests/control_data/graphical_units_data/geometry/CircleTest.npz``).
+To create the control data for your test, you have to use the flag ``--set_test`` along with pytest. 
+For the example above, it would be ``pytest test_geometry.py::test_circle --set_test -s``. (``-s`` is here to see manim logs, so you can see what's going on). 
```

Comment:
I think having having a `.. code-block::bash` with the input needed would make this more explicit/noticeable

Reply:
Will do, thanks